Error in connecting to the VLO due to name mismatch

The send_info command (in healthcheck) causes the following error 
Caused by: java.net.URISyntaxException: Illegal character in path at index 122: http://catalog.clarin.eu/oai-harvester/ILC_CNR_for_CLARIN_IT_repository_hosted_at_Institute_for_Computational_Linguistics_"A._Zampolli"_National_Research_Council_in_Pisa.html

The correct URI at vlo for our institute is 
https://vlo.clarin.eu/data

/ILC_CNR_for_CLARIN_IT_repository_hosted_at_Institute_for_Computational_Linguistics_A_Zampolli_National_Research_Council_in_Pisa.html
With no quotation and fullstop.

--- a/dspace-api/src/main/java/cz/cuni/mff/ufal/health/VLOCheck.java
+++ b/dspace-api/src/main/java/cz/cuni/mff/ufal/health/VLOCheck.java
@@ -46,7 +46,7 @@ public class VLOCheck extends Check {
                 return "PLEASE configure lr.harvester.info.url";
             }
             String dspace_name = ConfigurationManager.getProperty("dspace.name");
-            dspace_name = dspace_name.replaceAll("[() ,/-]+","_");
+            dspace_name = dspace_name.replaceAll("[() ,/.\"-]+","_");
             dspace_name = StringUtils.stripAccents(dspace_name);
             harvesterInfoUrl = harvesterInfoUrl.endsWith("/") ? harvesterInfoUrl + dspace_name :
                     harvesterInfoUrl + "/" + dspace_name;

